website/content/docs/commands/job/tag.mdx

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+page_title: 'Commands: job tag'
+description: |
+  Use the job tag commands to manage versions. The job tag apply command saves a job version tag, and the job tag unset command removes a job version tag.
+---
+
+
+# Command: job tag
+
+Use the `job tag` command to manage tags for job versions. `job tag` has the
+following subcommands:
+
+- [`job tag apply`](#apply): Save a job version tag.
+- [`job tag unset`](#unset): Remove a tag from a job version.
+
+## Usage
+
+```shell
+nomad job tag <subcommand> [options] [args]
+```
+
+## General options
```

Comment:
I don't know if it's typical of our CLI docs, but I might consider moving the General Options to the bottom of this page. These are less specifically relevant than the `tag apply` and `tag unset` information and as a user I don't think I'd expect to see them first.

Reply:
I don't like where they are either, but the other CLI pages have general options before specific command options. Something to change across all pages in the future.